NEWS.md

```diff
@@ -7,6 +7,12 @@
   when a vector of duplicate indices is passed when doing column selection
   ([#3302](https://github.com/JuliaData/DataFrames.jl/pull/3302))
 
+* Sorting related functions (`issorted`, `sort`, `sort!` and `sortperm`) have a
```

Comment:
* No newline should be added before `*`.
* The description should be shorter (it stays in NEWS.md). If users want details they can click the link to PR.
* Link to PR should be added (see other entries for examples)